tls/s2n_cipher_preferences.h

```diff
@@ -23,6 +23,7 @@
 struct s2n_cipher_preferences {
     uint8_t count;
     struct s2n_cipher_suite **suites;
+    bool prioritize_chacha20;
```

Comment:
I'm not sure if `prioritize` is the right word here... It's kinda vague without context around the feature. Do we have any other ideas on what to call it (I'll think about it as well)

Reply:
Maybe something like `allow_client_chacha20_priority`?

Reply:
I got the inspiration from openssl's `SSL_OP_PRIORITIZE_CHACHA` but I can see how it's confusing. How about `allow_chacha20_priority` or `allow_chacha20_boosting`?

Reply:
I'm fine with `allow_chacha20_boosting`